app/models/setting/ansible.rb

```diff
@@ -15,20 +15,29 @@ def load_defaults
             set(
               'ansible_port',
               N_('Use this port to connect to hosts '\
-                 'and run Ansible. You can override this on hosts'\
-                ' by adding a parameter "ansible_port"'),
+                 'and run Ansible. You can override this on hosts '\
+                 'by adding a parameter "ansible_port"'),
               22,
               N_('Port')
             ),
             set(
               'ansible_user',
-              N_('Foreman will try to connect to hosts as this user by default'\
-                 ' when running Ansible playbooks. You can override this '\
-                 ' on hosts by adding a parameter "ansible_user"'),
+              N_('Foreman will try to connect to hosts as this user by '\
+                 'default when running Ansible playbooks. You can '\
+                 'override this on hosts by adding a parameter '\
+                 '"ansible_user"'),
               'root',
               N_('User')
             ),
             set(
+              'ansible_become',
+              N_('Foreman will use the sudo command to run roles on hosts '\
+                 'You can override this on hosts by adding a parameter '\
+                 '"ansible_become"'),
+              'True',
```

Comment:
try replacing `'True'` with `true`, our helper should recognize the boolean and use dropdropdown instead of text field